mcux/devices/MK22FA12/fsl_device_registers.h

```diff
@@ -0,0 +1,64 @@
+/*
+ * The Clear BSD License
```

Comment:
The Zephyr project decided not to accept this license upstream. We need to see if NXP can release an updated MCUXpresso SDK for this SoC.